src/order/countable_dense_linear_order.lean

```diff
@@ -0,0 +1,288 @@
+/-
+Copyright (c) 2020 David Wärn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Wärn
+-/
+import order.generic_cofilter
+import data.finset
+
+noncomputable theory
+open_locale classical
+
+section cmp
```

Comment:
I think everything in this section belongs elsewhere - possibly where `cmp` or `ordering` is defined